### ISSUE TITLE
Fantastic, full-featured, but... child components don't update.

### ISSUE THREAD

**Issue by planetpearce** [maxmarinich/react-alice-carousel#83]
I successfully built a prototype using a custom Slide component to populate items:

````
<AliceCarousel>
   <Slide title={slide1.title} ... />
   <Slide title={slide2.title} ... />
</AliceCarousel>
````
Changing a property value on a slide component does not fire any lifecycle events for that component. If I move the `<Slide>` outside of `<AliceCarousel>` it fires events and updates just fine. Everything that I've tried to force a `<Slide>` to re-render hasn't worked.

Looking in the source, it appears you clone the children/items which might explain why property updates have not effect. 

In summary:
- Am I missing something? I certainly hope so since it's a great library that is otherwise wonderful.
- Or is there a workaround where I can force the clones to be updated?
- Or... I don't mind diving into the source if you have any thoughts on how to approach a solution.

Again, apologies if I missed something.

**Comment by maxmarinich** [comment]
Hi, @planetpearce! Please, provide a full code sample(like [here](https://codesandbox.io/s/child-components-example-imovd)).

**Comment by nolakat** [comment]
Hi @maxmarinich I actually came here because I am having the same issue. Currently my code looks something like this. 
`
<AliceCarousel
        autoHeight = {true}
        mouseDragEnabled >

        { this.state.sortedReviews.map((reviewGroup,index) => (
                <div key={index} className="review-group">
                    {reviewGroup.map((review, index) =>(
                            <div key={index} review-id={review.id}>
                                <Review {...review} key={index} />
                            </div>
                        ))
                    }
                </div>
            ))}
    </AliceCarousel>`

Unfortunately when the state of 'Sorted Reviews' updates it is not reflected in the slider. It also appears that there is not an option to update/destroy/rebuild the slider as well. Is this correct? Thank you!

**Comment by nolakat** [comment]
@maxmarinich I posted an update with a clearer example here: 

https://codesandbox.io/s/child-components-example-oqt5j?fontsize=14

**Comment by maxmarinich** [comment]
@planetpearce, @nolakat ! It's happening, because the `AliceCarousel` is a PureComponent. Like i've noticed in the docs - preferable to use `items` property instead of children. See [example](https://codesandbox.io/s/child-components-example-wqh7u). In one of the next releases I will improve the behavior. Thanks for the feedback.
